src/Assistants/InvoiceAssistant.php

```diff
@@ -0,0 +1,423 @@
+                                    'defaultValue' => 0,
+                                    'options' => [
+                                        'name' => 'invoiceAssistant.maximumAmount',
+                                    ],
+                                ],
+                            ],
+                        ],
+
+                        [
+                            "title" => 'invoiceAssistant.showDesignatedUseTitle',
+                            "form" => [
+                                "showDesignatedUse" => [
+                                    'type' => 'toggle',
+                                    'defaultValue' => true,
+                                    'options' => [
+                                        'name' => 'invoiceAssistant.showDesignatedUse'
+                                    ]
+                                ],
+                            ],
+                        ],
+                        [
+                            "title" => 'invoiceAssistant.designatedUseTitle',
+                            "condition" => 'showDesignatedUse',
+                            "description" => 'invoiceAssistant.designatedUseDescription',
+                            "form" => [
+                                "designatedUse" => [
+                                    'type' => 'text',
+                                    'defaultValue' => "%s",
+                                    'options' => [
+                                        'name' => 'invoiceAssistant.designatedUse',
+                                    ],
+                                ],
+                            ],
+                        ],
+                    ]
+                ]
+            ]
+        ];
+    }
+
+    /**
+     * @return string
+     */
+    private function getLanguage()
+    {
+        if ($this->language === null) {
+            $this->language =  \Locale::getDefault();
+        }
+
+        return $this->language;
+    }
+
+    /**
+     * @return string
+     */
+    private function getIcon()
+    {
+        $app = pluginApp(Application::class);
+        $icon = $app->getUrlPath('invoice').'/images/icon.png';
+
+        return $icon;
+    }
+
+    private function getMainWebstore(){
+        if($this->mainWebstore === null) {
+            /** @var WebstoreRepositoryContract $webstoreRepository */
+            $webstoreRepository = pluginApp(WebstoreRepositoryContract::class);
+
+            $this->mainWebstore = $webstoreRepository->findById(0)->storeIdentifier;
+        }
+        return $this->mainWebstore;
+    }
+
+    /**
+     * @return array
+     */
+    private function getWebstoreListForm()
+    {
+        if($this->webstoreValues === null)
+        {
+            $webstores = $this->webstoreRepository->loadAll();
+            /** @var Webstore $webstore */
+            foreach ($webstores as $webstore) {
+                $this->webstoreValues[] = [
```

Comment:
Du benutzt `$this->webstoreValues` als Array ohne es vorher als Array initialisiert zu haben.